zoomable-image/sub-sampling-image/src/commonMain/kotlin/me/saket/telephoto/subsamplingimage/internal/tileGridGenerator.kt

```diff
@@ -65,6 +65,28 @@ internal fun ImageRegionTileGrid.Companion.generate(
   )
 }
 
+internal fun ImageRegionTileGrid.Companion.defaultMinSize(
```

Comment:
While this looks great, do you not need the calculation to happen for every zoom level here?

https://github.com/saket/telephoto/blob/27128064ce3c1107dda5b6b8fa301f37e48eef28/zoomable-image/sub-sampling-image/src/commonMain/kotlin/me/saket/telephoto/subsamplingimage/internal/tileGridGenerator.kt#L30-L32